client/src/app/layout/SidebarApp/SidebarApp.tsx

```diff
@@ -30,44 +30,25 @@ export const SidebarApp: React.FC = () => {
   const { t } = useTranslation();
   const { search } = useLocation();
   const history = useHistory();
-  enum PersonaKey {
-    ADMINISTRATION = "Administration",
-    MIGRATION = "Migration",
-  }
+  const PersonaKey = {
+    ADMINISTRATION: "Administration",
+    MIGRATION: "Migration",
+  };
 
-  const options = [
-    <SelectOption
-      key="dev"
-      component="button"
-      value={PersonaKey.MIGRATION}
-      isPlaceholder
-    >
-      {PersonaKey.MIGRATION}
-    </SelectOption>,
-    ...(adminAccess
-      ? [
-          <SelectOption
-            key="admin"
-            component="button"
-            value={PersonaKey.ADMINISTRATION}
-          >
-            {PersonaKey.ADMINISTRATION}
-          </SelectOption>,
-        ]
-      : []),
+  const personaOptions: SelectOptionProps[] = [
+    { value: PersonaKey.MIGRATION, children: PersonaKey.MIGRATION },
   ];
-
-  const personaOptions: string[] = [
-    PersonaKey.MIGRATION,
-    PersonaKey.ADMINISTRATION,
-  ];
-
-  const [selectedPersona, setSelectedPersona] =
-    useLocalStorage<PersonaKey | null>({
-      key: LocalStorageKey.selectedPersona,
-      defaultValue: null,
+  adminAccess &&
+    personaOptions.push({
```

Comment:
I would push this up a few lines, have it inside the `personaOptions` array and add a `.filter(Boolean)` at the end...

```ts
const personaOptions: SelectOptionProps[] = [
  { value: PersonaKey.MIGRATION, children: PersonaKey.MIGRATION },
  adminAccess && { value: PersonaKey.ADMINISTRATION, children: PersonaKey.ADMINISTRATION },
].filter(Boolean);
```